tools/testing/test_all.bat

```diff
@@ -188,7 +188,7 @@ do
   fi
 
   if [ $RunCodeCoverage == "true" ]; then
-    find ./libraries -type f -name "*.cpp" -exec gcov {} \; &> /dev/null
+    find ./src/libraries -type f -name "*.cpp" -exec gcov {} \; &> /dev/null
```

Comment:
This is the part that needs to change to analyze different subfolders/projects in each report.